tinker/admin/redirects/redirects_controller.py

```diff
@@ -62,8 +62,8 @@ def search_db(self, search_type, term):
         results.sort()
         return results
 
-    def delete_row_from_db(self, from_path):
-        redirect_to_delete = BethelRedirect.query.get(from_path)
+    def delete_row_from_db(self, id):
+        redirect_to_delete = BethelRedirect.query.get(id)
```

Comment:
It might be worth trying to call out the column in the `BethelRedirect.query.get()` call directly.

Something like `BethelRedirect.query.get(BethelRedirect.id == id)`

Reply:
This didn't work. It stopped deleting the redirect. I kept it the way it was for now.